terraform/gce-vm/startup-script.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
+set -o errexit
+
+# Update and upgrade APT reop
+export DEBIAN_FRONTEND=noninteractive
+apt-get -y update
+apt-get -y upgrade
+
+echo "Installing prerequisites..."
+if ! command -v "kubectl" >/dev/null 2>&1; then
+    echo "Installing kubectl..."
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+    install -o root -g root -m 0755 kubectl /usr/local/bin/kubectl
+else
+    echo "kubectl already been installed. skip..."
+fi
+
+if ! command -v "gcloud" >/dev/null 2>&1; then
+    echo "Installing gcloud..."
+    apt-get install -y apt-transport-https ca-certificates gnupg
+    echo "deb [signed-by=/usr/share/keyrings/cloud.google.gpg] https://packages.cloud.google.com/apt cloud-sdk main" | tee -a /etc/apt/sources.list.d/google-cloud-sdk.list
+    curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | apt-key --keyring /usr/share/keyrings/cloud.google.gpg add -
+    apt-get update -y && apt-get install -y google-cloud-cli
+    echo "gcloud installation completed"
+else
+    echo "gcloud already been installed. skip..."
+fi
```

Comment:
Can't we use the containerized versions for which we already have shell functions defined?

Reply:
Unfortunately, we can not use docker on this env because later anthos setup step will install container runtime by own. However there are few thoughts came out to mind that I can improve this step. Will revisit on next PR.